Makefile

```diff
@@ -333,6 +335,9 @@ build-windows: .prebuild prepare-enterprise
 
 build: build-linux build-windows build-osx
 
+build-current: .prebuild prepare-enterprise
```

Comment:
People should be doing `make run`, but our `make run` doesn't follow the regular pattern where `make` just spits out an executable. Since they're just doing `make` in some cases, I thought I'd try to emulate how that normally works. Thoughts?